### ISSUE TITLE
Ensure Web Accessibility

### ISSUE THREAD

**Issue by pancakereport** [data-8/textbook#213]
Stricter guidelines will be in place April 2026 for all Berkeley managed sites. [See specifics here](https://dap.berkeley.edu/ada-title-ii-updates/title-ii-ada-what-does-it-mean-us).

#204 Created a workflow that monitors accessibility violations, but includes no fixes.

**Comment by pancakereport** [comment]
We're passing the automated accessibility tests now through improvements to the underlying software (Jupyter Book) and the context of this book, including a complete audit of the alt text quality. If you find an accessibility issue with this textbook please let us know by filing a new issue!
